tests/datasource/fluent/test_sql_datasources.py

```diff
@@ -1,19 +1,23 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Sequence, Tuple
```

Comment:
Same note - you don't need `Tuple` when you can use `tuple`

Reply:
@cdkini Suggestion incorporated/committed -- thanks!

Reply:
I still see `Tuple` here and used in type hints